src/js/schemas/service-schema/ContainerSettings.js

```diff
@@ -58,7 +62,12 @@ const ContainerSettings = {
       }
     },
     parameters: {
-      title: 'Parameters',
+      title: 'Docker Parameters',
+      description: (
+        <span>
+        Supply options for the <a href="https://docs.docker.com/engine/reference/commandline/run/">docker run</a> command executed by the Mesos containerizer.
```

Comment:
We should also add `target="_blank"` to this link, as such:
`<a href="https://docs.docker.com/engine/reference/commandline/run/" target="_blank">`